doc/parameters.md

```diff
@@ -517,6 +517,13 @@ Sets the maximum number and size of buffers used for reading large client reques
 The format for this value is defined by the [`large_client_header_buffers` NGINX
 directive](https://nginx.org/en/docs/http/ngx_http_core_module.html#large_client_header_buffers)
 
+### `APICAST_POLICY_BATCHER_SHARED_MEMORY_SIZE`
```

Comment:
It would be nice to have info about units that can be used. Maybe a reference to https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#lua_shared_dict ??

Reply:
fixed